usvm-python/usvm-python-main/src/main/kotlin/org/usvm/machine/types/VirtualTypes.kt

```diff
@@ -40,7 +41,7 @@ data class ConcreteTypeNegation(val concreteType: ConcretePythonType) : VirtualP
     }
 }
 
-sealed class TypeProtocol : VirtualPythonType() {
+sealed class TypeProtocol(private val slotId: SlotId? = null) : VirtualPythonType() {
```

Comment:
This is a temporary solution, since some slots, such as `nb_int`, `nb_index` and `mp_length`, are missing their implementation in `virtual_objects.c`.

Reply:
Ok. Please, add a comment about that

Reply:
Already done it